src/test/java/core/basesyntax/service/impl/FruitTransactionProcessorImplTest.java

```diff
@@ -0,0 +1,124 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.MapStorage;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.strategy.OperationHandlerStrategy;
+import core.basesyntax.service.strategy.impl.BalanceOperationHandler;
+import core.basesyntax.service.strategy.impl.OperationHandlerStrategyImpl;
+import core.basesyntax.service.strategy.impl.PurchaseOperationHandler;
+import core.basesyntax.service.strategy.impl.ReturnOperationHandler;
+import core.basesyntax.service.strategy.impl.SupplyOperationHandler;
+import java.util.List;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionProcessorImplTest {
+    private FruitTransactionProcessorImpl fruitTransactionProcessor;
+    private Storage storage;
+
+    @BeforeEach
+    void setUp() {
+        storage = new MapStorage(); // Use your preferred storage implementation here
+        FruitService fruitService = new FruitServiceImpl(storage);
+        OperationHandlerStrategy strategy = new OperationHandlerStrategyImpl(

```

Comment:
Please move all code from this method to the method `@BeforeAll beforeAll()` as I described in comments above.
Fix all similar cases in all test classes where needed.